src/main/java/com/google/sps/data/PlanMailResponse.java

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.sps.utility.DateInterval;
+import java.util.*;
```

Comment:
Change this to `import java.util.List;`, the style guide [doesn't like wildcard imports](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports).

Reply:
`Resolved`